src/components/AccountLinks/AccountLinksTs.ts

```diff
@@ -35,14 +37,13 @@ export class AccountLinksTs extends Vue {
     @Prop({ default: '' }) link: string;
     @Prop({ default: '' }) icon: string;
 
-    public explorerBaseUrl: string;
     public faucetBaseUrl: string;
-
+    private networkType: NetworkType;
     /// region computed properties getter/setter
     /// end-region computed properties getter/setter
 
     public get explorerUrl() {
-        return this.explorerBaseUrl.replace(/\/+$/, '') + '/accounts/' + this.account.address;
+        return networkConfig[this.networkType].explorerUrl.replace(/\/+$/, '') + '/accounts/' + this.account.address;
```

Comment:
I'm a little concerned about regex usage in general across our application - in particular, this. Did you review the [SonarCloud flag](https://sonarcloud.io/organizations/symbol/rules?open=typescript%3AS5852&rule_key=typescript%3AS5852) for this?

Reply:
not really it didn't show up as a security hotspot in the merged PR https://github.com/symbol/desktop-wallet/pull/1784, but to me, it still looks like it's not much of an issue as it's not a user-controlled input neither the string URL is too long.

Reply:
I do. I'd not block this release, but open up a new issue, and include this with the refactor tag for the next release. 👍🏻

Reply:
`new Url('/accounts/' + this.account.address, networkConfig[this.networkType])` ?
(optionaly toString if needed)